Added pkg-config file.

kyucrane opened the pull request.

pfultz2 commented:
Oh awesome, thanks. A few things. The `CXX_EXTRA_FLAGS` shouldn't be in the package. Those flags are actually part of the toolchain, and only provided in the cmake for convenience when testing. Secondly, is the `URL` supposed to refer to the documentation or the repo?

kyucrane commented:
The pkg-config guide says "An URL where people can get more information about and download the package." .  I removed CXX_EXTRA_FLAGS and updated the URL to point to the repo.
